### ISSUE TITLE
Accessibility/Identity Providers: Unable to select a provider from the providers list using keyboard 

### ISSUE THREAD

**Issue by agagancarczyk** [keycloak/keycloak#18101]
### Describe the bug

Users can't select a provider for the first time from the providers list using the keyboard. 

It was tested in Chrome using Chromevox.

### Version

19.0.0

### Expected behavior

Users should be able to use tabs/arrows to select the provider. 

### Actual behavior

Users can't select a provider using the keyboard. 

### How to Reproduce?

- Go to Identity providers from the left-side menu
- Try to select any provider using tabs/arrows.
